cmd/start_server/main.go

```diff
@@ -15,9 +15,28 @@
 package main
 
 import (
+	"context"
+	"flag"
+	"log"
+
+	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/logger"
 	"github.com/googleinterns/cloudai-gcp-test-resource-reaper/pkg/manager"
 )
 
 func main() {
-	manager.StartServer("localhost", "8000")
+	port := flag.String("port", "8000", "port to run gRPC server on")
+	logsProject := flag.String("logs-project", "", "GCP Project ID for where to store logs")
+	logsName := flag.String("logs-name", "", "name of logs")
```

Comment:
Can we give this a default?

Reply:
I thought cloud logging should be optional, so if no value is set then there will be no cloud logging